src/main/docs/guide/xray/xrayRecorder.adoc

```diff
@@ -0,0 +1,32 @@
+The module `micronaut-aws-xray` produces two essential beans:
+
+* https://docs.aws.amazon.com/xray-sdk-for-java/latest/javadoc/com/amazonaws/xray/AWSXRayRecorderBuilder.html[com.amazonaws.xray.AWSXRayRecorderBuilder]
+* https://docs.aws.amazon.com/xray-sdk-for-java/latest/javadoc/com/amazonaws/xray/AWSXRayRecorder.html[com.amazonaws.xray.AWSXRayRecorder]
+
+The `AWSXRayRecorderBuilder` bean allows you to adjust the behaviour of the `AWSXRayRecorder`. Note that the beans of type https://docs.aws.amazon.com/xray-sdk-for-java/latest/javadoc/com/amazonaws/xray/plugins/Plugin.html[com.amazonaws.xray.Plugin] and https://docs.aws.amazon.com/xray-sdk-for-java/latest/javadoc/com/amazonaws/xray/listeners/SegmentListener.html[com.amazonaws.xray.SegmentListener] are automatically configured.
+
+For advanced configuration options that are not suitable to provide via `application.yml`, you can declare a
+`BeanCreatedEventListener` bean that listens for builder bean creations, and apply any further customisation there:
+
+```java
+@Singleton
+class XRayRecorderBuilderBeanListener implements BeanCreatedEventListener<AWSXRayRecorderBuilder> {
+
+    @Override
+    public AWSXRayRecorderBuilder onCreated(BeanCreatedEvent<AWSXRayRecorderBuilder> event) {
+        return event.getBean().withContextMissingStrategy(new LogErrorContextMissingStrategy());
+    }
+}
+```
+
+Finally you can configure https://docs.aws.amazon.com/xray/latest/devguide/xray-console-sampling.html[the sampling rule] definition file using the configuration property `aws.xray.samplingRule`.
```

Comment:
Micronaut configuration is kebap case. 

Instead of `aws.xray.samplingRule` it should be `aws.xray.sampling-rule`

Reply:
I have changed this.